src/main/java/com/game/Battleship/models/Ship.java

```diff
@@ -0,0 +1,46 @@
+package com.game.Battleship.models;
+import  java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Ship {
+
+    private String name;
+    private int size;
+    private List<Cell> shipBody;
+
+    public Ship(String name,int size){
+        this.name = name;
+        this.size = size;
+        this.shipBody = generateShipBody();
+    }
+
+    public String getName(){
+        return this.name;
+    }
+
+    public void setName(String name){
+        this.name = name;
+    }
+
+    public int getSize(){
+        return this.size;
+    }
+
+    public void setSize(int size){
+        this.size = size;
+    }
+
+    private List<Cell> generateShipBody(){
```

Comment:
Just out of curiosity, what is `shipBody` and how is it different from `size`?

Reply:
shipBody is an array of Cell objects and size will determine how many Cell objects a ship will have. I missed to add the getShipBody method.
<img width="422" alt="Screen Shot 2021-11-30 at 4 52 31 PM" src="https://user-images.githubusercontent.com/63215085/144243783-785bac92-2d21-40cb-8204-72c949ad2532.png">

Reply:
Gotcha 👍🏼  thanks for the explanation!